### PR TITLE
Log device information (8.x)

### PR DESCRIPTION
**Pull request by janpio**
Adds 3 log calls before handing over to `simctl` to do the actual launching.

Part of #254

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ios-sim",
-  "version": "8.0.1",
+  "version": "8.0.2-dev",
```
Comment (erisu) [inline package.json]: Is this change required?

If `-dev` is required, I think this should have been done after the `8.0.1` patch release so that people who submit PRs do not need to manage this.

I actually already merged in a PR into the `8.x` branch which released in `8.0.2` but the `package.json` still has `8.0.1` from that PR.

File: src/lib.js [hunk 2]
```diff
@@ -458,6 +458,11 @@ let lib = {
         // --devicetypeid is a string in the form "devicetype, runtime_version" (optional: runtime_version)
         let device = getDeviceFromDeviceTypeId(devicetypeid)
 
+        // log device information
+        console.log(util.format('device.name: %s', device.name))
```
Comment (erisu) [inline src/lib.js]: Do you think we should try to make these a bit more user friendlier since other tools also uses `ios-sim`?

For example: 'device.name: %s' become 'Device Name: %s'?

File: package.json (hunk 1 above)
Comment (janpio) [reply package.json]: I just followed what I saw in other commits on https://github.com/ios-control/ios-sim/commits/master I guess.

File: src/lib.js (hunk 2 above)
Comment (janpio) [reply src/lib.js]: Same here, just following more or less what was already part of the codebase - https://github.com/ios-control/ios-sim/blob/2e8cb284795bb3a656b6db59590c63f726e3e608/src/lib.js#L472. But the string really doesn't matter in this PR, it's about that this information is returned at all.

Comment (erisu) [reply src/lib.js]: 👍
